### ISSUE TITLE
Use better naming

### ISSUE THREAD

**Issue by malmans2** [MaceKuailv/seaduck#52]
These are some of the variables defined in `smart_func`:
`['ckbl', 'cksz', 'ckus', 'i', 'ix', 'j', 'k', 'n', 'npck', 'p', 'pr', 'pre', 'prs', 'sf']`

It's hard to follow when it's not obvious what they are.
Try to use better names, and use vowels: Luckily python variable names can be any length :)

**Comment by MaceKuailv** [comment]
Yeah, this is also true for all other modules. It is an arduous task that I have been postponing. I will get to this soon.

**Comment by MaceKuailv** [comment]
Did a lot of renaming in PR #64. I think except for the smart_read (which will be entirely replaced). The names looks alright. 

@malmans2 what do you think?

**Comment by MaceKuailv** [comment]
resolved with PR #64 and #65
